source/manual/receiving-emails-from-email-alert-api-in-integration-and-staging.html.md

```diff
@@ -6,32 +6,28 @@ layout: manual_layout
 parent: "/manual.html"
 ---
 
-In `integration` and `staging` Email Alert API defaults to sending emails
-to a single test address: `success@simulator.amazonses.com`. This is used to
-simulate a successful email sending.
+In `integration` and `staging` Email Alert API defaults to [writing emails to
+a logfile][logging-emails] rather than sending them to Notify. Only [email
+addresses specified in GOV.UK Puppet][puppet-config] will receive actual
+emails.
 
-However, you can override this for specific email addresses for testing
+You can override this for specific email addresses for testing
 purposes. To do this, [you will need to be added as a team member to
 the GOV.UK Email Integration or Staging service in Notify][add-in-notify] and
-make changes to [govuk-puppet].
+make changes to GOV.UK Puppet.
 
-[add-in-notify]: /manual/govuk-notify.html#receiving-emails-from-govuk-notify
-
-1. Add your email address to the override whitelist in the file [common.yml](https://github.com/alphagov/govuk-puppet/blob/master/hieradata_aws/common.yaml#L540-L568) (the list needs to be kept in sync with the team members list in Notify). This is set as an environment variable via hieradata under the key of
-   `govuk::apps::email_alert_api::email_address_override_whitelist`. It should look something like this:
-
-   ```
-    govuk::apps::email_alert_api::email_address_override_whitelist:
-      - your.name@digital.cabinet-office.gov.uk
-   ```
-
-2. Create a branch with your changes and push them to GitHub. Deploy the changes by running the [deploy-puppet](https://deploy.integration.publishing.service.gov.uk/job/Deploy_Puppet) job for the environment you're testing on, providing your branch name instead of a release tag.
-
-3. Once these changes have been deployed and the environment variable `EMAIL_ADDRESS_OVERRIDE_WHITELIST` is populated with your address you can test that you can receive emails by running the [support:send_test_email[name@example.com]](https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=email-alert-api&MACHINE_CLASS=email_alert_api&RAKE_TASK=support:send_test_email[your.name@digital.cabinet-office.gov.uk]) rake task.
+1. Add your email address to the `govuk_notify_recipients` list in
+   [hieradata_aws/common.yml][].
+2. Create a branch with your changes and push them to GitHub. Deploy the
+   changes by running the [deploy-puppet][] job for the environment you're
+   testing on, providing your branch name instead of a release tag.
+3. Once these changes have been deployed, and the environment variable
+   `GOVUK_NOTIFY_RECIPIENTS` is populated with your address, you can test that
+   you can receive emails by running the [Send a test email][] support task.
 
 ## Testing digest emails
```

Comment:
Possibly out-of-scope: I think we should just delete this, as it's just a poor substitute for how the digest process actually works, and it's not something we've needed to do.

Reply:
Yeah, I'm not convinced by this section. I'll plan to chop it by the end of the week if I manage to get the groups sorted.